db/seeds.rb

```diff
@@ -47,9 +47,9 @@
 ronald4 = User.find(4)
 
 # tenants
-Tenant.create!(
+bridge_builders = Tenant.create!(
```

Comment:
I would leave this in there. Real organizations will have spaces (and occasionally punctuation). The before_save will ensure that a slug is created.